configuration/jsr223.md

```diff
@@ -290,7 +290,7 @@ The `default` preset is preloaded, so it does not require importing.
 | `things`                | Instance of `org.openhab.core.thing.ThingRegistry`                                       |
 | `rules`                 | Instance of `org.openhab.core.automation.RuleRegistry`                                   |
 | `events`                | (internal) Used to send events, post commands, etc. [Details](#events-operations) below] |
-| `actions`               | Instance of `org.openhab.core.thing.binding.ThingActions`                                |
+| `actions`               | Instance of `org.openhab.core.automation.module.script.defaultscope.ScriptThingActions`  |
```

Comment:
```suggestion
| `actions`               | Use `actions.get(String bindingName, String thingUid)` to get the actions of a Thing (instanceof `org.openhab.core.thing.binding.ThingActions`)  |
```

Reply:
Please fix the table's formatting afterwards, you may use http://markdowntable.com for that.

Reply:
I’m convinced that knowing the type of the `actions` variable is useful for a typesafe coding.  

E.g. I’m using Intellij Idea with a custom `.gdsl` file to define types of context variables and imported constants to get useful code suggestions and checks.

I’ve noticed that all other table entries are containing types. The entry you suggested contains a way of use.  

As for me, it seems inconsistent. Maybe https://www.openhab.org/docs/configuration/actions.html would be a better place to explain how to use actions in rule scripts to the user.  

The same table from this doc has an `itemRegistry` entry without usage explanation and it seems fine. I don’t see how `actions` variable is different in that regard.

If you insist on having the usage description for the `actions` variable, I propose to combine both type definition and usage explanation.

Reply:
I've added the link you provided and formatted the table.

Please look if is that what you meant?